tests/main/snapctl/task.yaml

```diff
@@ -11,6 +11,7 @@ execute: |
     echo "Verify that snapctl -h runs without a context"
     if ! snapctl -h; then
         echo "Expected snapctl -h to be successful"
+        exit 1
```

Comment:
:man_facepalming: I _thought_ we had a test for it! Don't run a git blame.

Reply:
I won't, it stays between us ;)